index.html

```diff
@@ -242,48 +244,54 @@ <h4 class="drp_txt">CSS</h4>
+        <div id="dropdown_header">
+          <h3 class="drp_text">Frameworks</h3>
+          <i class="dropdown_hamburger">
+            <i class="fa-solid fa-angle-right"></i>
+          </i>
+        </div>
       </div>
-
       <hr class="hr"></hr>
-      <div id="dropdown_header">
-        <h3 class="drp_text">Skills</h3>
-        <i class="dropdown_hamburger">
-          <i class="fa-solid fa-angle-right"></i>
-        </i>
+      <div class="dropdown_container">
+        <div id="dropdown_header">
+          <h3 class="drp_text">Skills</h3>
+          <i class="dropdown_hamburger">
+            <i class="fa-solid fa-angle-right"></i>
+          </i>
+        </div>
       </div>
       <hr class="hr"></hr>
       
     </div>
   </section>
-  <section class="contact-section">
-    <div class="contactMe">
-      <div class="contactblock">
+
+  <section class="contactMe">
+    <div class="contactblock">
+    
         <h1 class="contactTxt">Contact me</h1>
-        <div class="Contact_body_text">
-          <p 
-            class="body-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
-          </p>
+      
+      
+      <div class="Contact_body_text">
+        <p 
+          class="body-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+        </p>
+      </div>
+      
+      <form class="	form" action="https://formspree.io/f/xnqrezql" method="post" >
+        <input type="text" id="name" name="user_name" placeholder="FullName"  required maxlength="30"/>
+        <input type="email" id="mail" name="user_email" placeholder="Email Address"  pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+.[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-])*$/" required />
+       
+        <textarea id="textarea" cols="30" rows="10" placeholder="Write your message here" name="user_message" required maxlength="500"></textarea>
+       
+        <div id="action">
+          <button type="submit" class="normalButtnAbout" onclick=""> Get in Touch</button>
         </div>
         
-        <form class="	form" action="https://formspree.io/f/xnqrezql" method="post" >
-          <input type="text" id="name" name="user_name" placeholder="FullName"  required maxlength="30"/>
-          <input type="email" id="mail" name="user_email" placeholder="Email Address"  pattern="/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+.[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-])*$/" required />
-         
-            <textarea id="textarea" cols="30" rows="10" placeholder="Write your message here" name="user_message" required maxlength="500"></textarea>
-         
-          <div id="action">
-            <button type="submit" class="normalButtnAbout" onclick=""> Get in Touch</button>
-          </div>
-          
-        </form>
```

Comment:
Unfortunately, the submit button of the form is at the left but according to the figma template, should be in the center. I suggest you move the button to the middle to make it consistent with the figma template. I suggest that you also check the border radius of the inputs and adjust them appropriately to fit the figma design

Your design: 
![image](https://user-images.githubusercontent.com/107795045/222868809-a0af05ab-7a5d-4cf5-95dd-66db5a4ef907.png)

Figma design: 
![image](https://user-images.githubusercontent.com/107795045/222868876-6fe705c5-90af-48c9-951e-dc1aa371caf4.png)